routes/mods.py

```diff
@@ -169,16 +186,19 @@ def dict_all(models):
         "tagline": fields.Str(validate=validate.Length(max=100)),
         "description": fields.Str(validate=validate.Length(max=10000)),
         "website": fields.Url(),
+        "is_private_beta": fields.Bool(),
         "authors": fields.List(fields.Nested(AuthorSchema)),
+        "playtesters": fields.List(fields.Str()),
         "status": EnumField(ModStatus),
         "icon": None
     }, locations=("json",))
-    async def patch_mod(self, mod_id: str = None, **kwargs):
+    async def patch_mod(self, mod_id: str = None, is_private_beta: bool = None, playtesters: List[str] = None, **kwargs):
         if not await Mod.exists(mod_id):
             abort(404, "Unknown mod")
 
         mod = await Mod.get(mod_id)
         updates = mod.update()
+        playtesters = await Playtesters.get(mod_id)
```

Comment:
Rename the variable so it doesn't conflict with the argument, and this function won't work how you want. Try:
```py
await Playtesters.query.where(Playtesters.mod_id = mod_id).gino.all()
```